src/brokers/add-broker/components/AddBrokerForm/YamlEditorView.tsx

```diff
@@ -48,19 +51,17 @@ const YamlEditorView: FC<YamlEditorViewProps> = ({
     onCreateBroker(yamlData);
   };
 
+  const onCancel = () => {
+    history.push('/k8s/all-namespaces/brokers');
+  };
+
   //event: contains information of changes
   //value: contains full yaml model
   const onChanges = (newValue: any, _event: any) => {
-    //yamlValue.yamlData = JSON.parse(newValue);
-    //instead of blatantly replce the whole contents
-    //compare them and only accept additive contents, or
-    // else warning and refuse to update.
-    console.log('old value:', fromState.cr);
-    console.log('new value', JSON.parse(newValue));
-    Object.assign(fromState.cr, JSON.parse(newValue));
-    //_.merge(yamlValue.yamlData, newValue);
-
-    console.log('module after merge:', fromState.cr);
+    dispatch({
+      operation: ArtemisReducerOperations.setModel,
+      payload: Object.assign(fromState.cr, JSON.parse(newValue)),
```

Comment:
I think just sending `JSON.parse(newValue)` in the payload should be sufficient.